### DISCUSSION TITLE
Multiple selection?

### DISCUSSION THREAD

**Discussion by rodhash** [adoyle-h/lsp-toggle.nvim#2] [General]
Super nice plugin man

Now just wondering, we we can do multiple selection and disable 2 or more sources at once? for me it seems to work only single selection

Thanks

**Comment by adoyle-h** [comment]
Please upgrade this plugin to latest version. This issue fixed in c1d9fd9dfe4f85141fc373eab39db310a30faa24 and fa92519bc413600d2ca7687fbff0a7b018adee50 .
The multiple selection only supported when telescope enabled.
Thank you.

**Comment by rodhash** [comment]
Hmm that's interesting, the multiple selection works only when calling `ToggleLSP` by the telescope extension `find pickers`.

I do have Telescope enabled but if I simply call `ToggleLSP` directly as a command `:ToggleLSP`, the multiple selection doesn't work.

Is this expected?

Anyway thanks for the tip, awesome plugin and now having multiple selection helps a lot.

**Reply by adoyle-h** [reply]
Thanks for pointing out the problem, it was my oversight. It gets fixed on 501d647. Please try again.

**Reply by rodhash** [reply]
Great, it's working fine now by calling it directly (or by keybindings). Thanks.
